src/commands/QuotesCommand/quotes.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I don't think this file should be on the repo.
This may lead to conflicts if everyone has a different version of it